drivers/hmis/app/graphql/types/hmis_schema/mutation_type.rb

```diff
@@ -64,6 +64,8 @@ class HmisSchema::MutationType < Types::BaseObject
     field :create_form_definition, mutation: Mutations::CreateFormDefinition
     field :update_form_definition, mutation: Mutations::UpdateFormDefinition
     field :delete_form_definition, mutation: Mutations::DeleteFormDefinition
+    field :publish_form, mutation: Mutations::PublishForm
+    field :create_draft_form, mutation: Mutations::CreateDraftForm
```

Comment:
I don't see a meaningful difference between when we use the work `form_definition` vs `form`. Maybe we can use `PublishFormDefinition` insetad of `PublishForm` for the new one.

It's also unclear from the names whats different between `CreateFormDefinition` (brand new form definition with no content) and `CreateDraftForm` (dup an existing form into a draft).

Maybe CreateDraftForm can be called `CreateNextDraftFormDefinition` or something?